package.json

```
@@ -59,29 +59,28 @@
     "@oclif/command": "^1.0",
     "@oclif/config": "^1.12.11",
     "@oclif/plugin-help": "^2.0",
-    "@stoplight/json": "^2.1.0",
-    "@stoplight/json-ref-resolver": "^1.5.0",
-    "@stoplight/types": "^7.0.1",
-    "@stoplight/yaml": "^2.3.0",
+    "@stoplight/json": "2.x.x",
+    "@stoplight/json-ref-resolver": "1.5.x",
+    "@stoplight/types": "7.x.x",
+    "@stoplight/yaml": "2.x.x",
```

I believe we'll want to switch to `^2.3.0`?
```suggestion
    "@stoplight/yaml": "^2.3.0",
```

---

Why? Only reason to do this is if we release breaking changes in stoplight/yaml without bumping the major version, which we should try not to do...